esmvaltool/diag_scripts/autoassess/land_surface_soilmoisture/soilmoisture.py

```diff
@@ -117,23 +173,67 @@ def land_sm_top(run):
         dff = vol_sm1_run - ecv_clim
 
         # save output and populate metric
-        iris.save(dff, os.path.join(run['dump_output'],
-                                    'soilmoist_diff_{}.nc'.format(season)))
-        name = 'soilmoisture MedAbsErr {}'.format(season)
-        dffs = dff.data
-        dffs = np.ma.abs(dffs)
-        metrics[name] = float(np.ma.median(dffs))
-
-    # record provenance
-    plot_file = "Autoassess soilmoisture metrics"
-    caption = 'Autoassess soilmoisture MedAbsErr for {}'.format(str(seasons))
-    provenance_record = get_provenance_record(caption, run)
-    cfg = {}
-    cfg['run_dir'] = run['out_dir']
-    # avoid rewriting provenance when running the plot diag
-    if not os.path.isfile(os.path.join(cfg['run_dir'],
-                                       'diagnostic_provenance.yml')):
-        with ProvenanceLogger(cfg) as provenance_logger:
-            provenance_logger.log(plot_file, provenance_record)
+        caption = f"Model minus CCI soil moisture climatology for {season}"
```

Comment:
You may want to mention which Model is used in the caption because this is currently not clear. See also the output generated here: https://esmvaltool.dkrz.de/shared/esmvaltool/esmvalbot-output/autoassess_soilmoisture_obs_fly-lj2pb243/recipe_autoassess_landsurface_soilmoisture_20230619_102113/ (under Data Files)